library/alloc/src/lib.rs

```diff
@@ -158,6 +158,7 @@
 #![feature(rustc_allow_const_fn_unstable)]
 #![feature(rustc_attrs)]
 #![feature(staged_api)]
+#![feature(strict_provenance)]
```

Comment:
This is a library feature, right? Language and library `#![feature]` here are separated. The library features have their own section above (line 88).

Reply:
![image](https://user-images.githubusercontent.com/1136864/160712378-9cdddf69-3320-40f0-a359-12301461d416.png)

this sorting ambiguity is going to haunt me for the rest of my life